### PR TITLE
README updates

### LINKED ISSUE #20: moab-vers: add method to get latest version (for online, exploded objects)
(sometimes called "current version")

We didn't find any code existing in the moab gem that is as simple as looking at the directory of an online moab object and parsing the name of the highest lexical sorting `vnnn` directory into an integer.

What we have now in PCC:

```
 version_metadata_file = Stanford::StorageServices.version_metadata(params['id'])
    vm = Moab::VersionMetadata.parse(version_metadata_file.read)
current_version: vm.versions.last.version_id
```

It may be (probably is) more performant to do this with Dir.entries.  See sul-dlss/preservation_core_catalog#24

### LINKED ISSUE #24: moab_vers: add gem release badge

### LINKED ISSUE #25: moab_vers: add gemnasium
(and badge)

### PR DESCRIPTION
**Pull request by ndushay**
update README:
- add usage information (what we know so far)
- convert to md from rdoc
- add badges for gemnasium and gem release version

resolves #20 
resolves #24 
resolves #25

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,109 +1,109 @@
...
+  require 'moab'
+```
 
-http://rubydoc.info/github/sul-dlss/moab-versioning/master/frames
+#### Configuration
 
-{rdoc-image:https://www.omniref.com/ruby/gems/moab-versioning.png}[https://www.omniref.com/ruby/gems/moab-versioning]
+```ruby
+Moab::Config.configure do
+  storage_roots 'yer_first_storage_root_dir_except_last_part_of_path'
```
Comment (jmartin-sul) [inline README.md]: what about something like:
`storage_roots [yer_first_storage_root_dir_except_last_part_of_path, second_storage_root_dir, etc]`

real-world usage example:
```ruby
storage_roots ["#{SDR_HOME}/sdr_repository/store01", "#{SDR_HOME}/sdr_repository/store02"]
```
(from https://github.com/sul-dlss/shared_configs/blob/sdr-preservation-core_stage/config/environments/stage.rb#L27)

File: README.md [hunk 2]
```diff
@@ -1,109 +1,109 @@
...
-      * {FileInstance} [1..*] = filepath and timestamp of any physical file having that signature
+#### Get Latest Version number
 
-* <b>{SignatureCatalog} = lookup table containing a cumulative collection of all files ever ingested</b>
-  * {SignatureCatalogEntry} [1..*] = an row in the lookup table containing storage information about a single file
-    * {FileSignature} [1] = file fixity information
+```ruby
+current_version = Stanford::StorageServices.current_version('oo000oo0000') # where oo000oo0000 is the druid
+expect(current_version).to be_an_instance_of Integer
+```
```
Comment (jmartin-sul) [inline README.md]: seems like this is a repeat of the above "Get Latest Version number" section?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+```
+  require 'moab'
+```
+
+#### Configuration
+
+```ruby
+Moab::Config.configure do
+  storage_roots 'yer_first_storage_root_dir_except_last_part_of_path'
```
Comment (jmartin-sul) [inline README.md]: what about something like:
`storage_roots [yer_first_storage_root_dir_except_last_part_of_path, second_storage_root_dir, etc]`

real-world usage example:
```ruby
storage_roots ["#{SDR_HOME}/sdr_repository/store01", "#{SDR_HOME}/sdr_repository/store02"]
```
(from https://github.com/sul-dlss/shared_configs/blob/sdr-preservation-core_stage/config/environments/stage.rb#L27)

File: README.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+- when path_method is `:druid_tree`, expect this directory structure: 'jq/937/jp/0017/jq937jp0017'
+- when path_method is `:druid`, expect this directory structure: 'jq937jp0017'
+
+#### Get Latest Version number
+
+```ruby
+current_version = Stanford::StorageServices.current_version('oo000oo0000') # where oo000oo0000 is the druid
+expect(current_version).to be_an_instance_of Integer
+```
```
Comment (jmartin-sul) [inline README.md]: seems like this is a repeat of the above "Get Latest Version number" section?

Comment (ndushay) [reply README.md]: except it says `Stanford::StorageServices` -- heh.   Which can matter for the druid tree thingy

Comment (jmartin-sul) [reply README.md]: ah, yes, i missed that, sorry!

**Pull request merged**